treefmt.nix

```diff
@@ -11,6 +11,19 @@
     "LICENSE"
   ];
 
+  formatter.actionlint = {
+    command = lib.getExe pkgs.actionlint;
+    includes = [
+      ".github/workflows/*.yml"
+      ".github/workflows/*.yaml"
```

Comment:
Could we also include `.github/actions/**/action.yml`?

Reply:
To be frank, I completely forgot about that one. I initially assumed `programs.actionlint.enable` from `treefmt-nix`, but then I saw we weren't using `treefmt-nix` so I just copied the `includes` that they had... Good catch, thanks!

Reply:
> Actionlint doesn't seem to support `.github/actions`:

Indeed: https://github.com/rhysd/actionlint/issues/46 😢 

Incidentally, looking through their issues there are a lot of missing features. Including the recently added support for ymal anchors: https://github.com/rhysd/actionlint/issues/133

And a question regarding whether the project is even maintained: https://github.com/rhysd/actionlint/issues/563

There are also other action linting projects, but I've never used them so IDK if they're any better. I suspect this will be fine for the most part and switching to another (or adding additional linters) will be simple enough in the future.